FiveCalls/FiveCalls/IssueListItem.swift

```diff
@@ -11,24 +11,28 @@ import SwiftUI
 struct IssueListItem: View {
     let issue: Issue
     let contacts: [Contact]
-    
+    @Environment(\.dynamicTypeSize) private var dynamicTypeSize
+
+    func usingRegularFonts() -> Bool {
```

Comment:
we could probably build a static typeutil for `usingRegularFonts` since it's being defined in multiple places

Reply:
Good in theory, but more complicated than I expected. I did try a few different ways, but getting runtime warnings about the environment variable.